app/models/audit_record.rb

```diff
@@ -7,6 +7,7 @@ class AuditRecord < ApplicationRecord
   TYPE_DROPOUT_STUDENT = 'dropout_student'
   TYPE_MERGE_USER_ACCOUNTS = 'merge_user_accounts'
   TYPE_UPDATE_EMAIL = 'update_email'
+  TYPE_UPDATE_NAME = 'update_name'
```

Comment:
Should we convert this into an enum? @mahesh-krishnakumar

Reply:
Yes, enum makes it clean to avoid a separate validation for `audit_type`